include/utils/mapvector.h

```diff
@@ -20,6 +20,9 @@
 #ifndef LIBMESH_MAPVECTOR_H
 #define LIBMESH_MAPVECTOR_H
 
+// libMesh Includes   -------------------------------
+#include <libmesh/pool_allocator.h>
```

Comment:
We usually use the `""` style of include for libmesh headers. I know there are subtle distinctions between `""` and `<>`, any particular reason for using the latter here?

Reply:
Whoops!  Originally I was trying to use Boost's pool allocator directly, before I discovered that Ben had already made a wrapper for it, and I guess I wasn't thinking about the quoting type when I changed that.  I'll fix it.